classes/ClaytonTestClass3.cls

```diff
@@ -0,0 +1,16 @@
+public class ClaytonTestClass3 {
+    public void AWrongMethodN_a_me_incredible_wow(){
```

Comment:
Methods should use the `lowerCamelCase` naming convention. Please rename accordingly.